Detectors/YOLO/yolo.py

```diff
@@ -23,17 +23,49 @@ def detect(self, image):
         Returns a list of detections where each item in list contains [x1, y1, x2, y2, conf]
         """
 
-        width = darknet.network_width(self.model)
-        height = darknet.network_height(self.model)
-        darknet_image = darknet.make_image(width, height, 3)
+        self.width = darknet.network_width(self.model)
+        self.height = darknet.network_height(self.model)
+        darknet_image = darknet.make_image(self.width, self.height, 3)
         image_rgb = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
-        image_resized = cv2.resize(image_rgb, (width, height),
+        image_resized = cv2.resize(image_rgb, (self.width, self.height),
                                 interpolation=cv2.INTER_LINEAR)
         darknet.copy_image_from_bytes(darknet_image, image_resized.tobytes())
         detections = darknet.detect_image(self.model, self.class_names, darknet_image)
         dect_list=[]
         for dect in detections:
-            dect_list.append(dect[2])
-        res = darknet.draw_boxes(detections, image, self.colors)
-        cv2.imshow("result",res)
+            if(dect[0]== 'person'):
+                dect_list.append(dect[2])
         return dect_list
+
+    def draw_box(self, detections, image):
+        for bbox, id in detections:
+            left, top, right, bottom = self.convert4cropping(image,bbox)
+            cv2.rectangle(image, (left, top), (right, bottom), [255,0,0], 1)
+            cv2.putText(image, f"id : {id} ", (left, top - 5), cv2.FONT_HERSHEY_SIMPLEX, 0.5, [0, 255, 0], 2)
+        return image
```

Comment:
Can create a new folder name `utility` and put this into the new python file since we will be requiring this for different detection model and it have no relation to `yolo` class

Reply:
Will be done in another PR